anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
We should probably leave the ignore tags in the file.  The user is supposed to enable the tests one by one as they get them to pass.